include/deal.II/lac/affine_constraints.h

```diff
@@ -704,7 +704,10 @@ class AffineConstraints : public Subscriptor
   add_lines(const IndexSet &lines);
 
   /**
-   * Add an entry to a given line. The list of lines is searched from the back
+   * Add an entry to a given line. In other words, this function adds
+   * a term $a_{ij} x_j$ to the constraints for the $i$th degree of freedom.
+   *
+   * This function searches existing constraints from the back
    * to the front, so clever programming would add a new line (which is pushed
    * to the back) and immediately afterwards fill the entries of that line.
    * This way, no expensive searching is needed.
```

Comment:
Unrelated to this PR, but this comment seems outdated, as we have a lines cache since at least 2008 or 2009 (not sure if it was you @bangerth or me to add it back then):
https://github.com/dealii/dealii/blob/ad29f4467f74f97dbb490a59bbd45ef78e629080/include/deal.II/lac/affine_constraints.h#L2003-L2013
Would you mind removing this paragraph?